components/Navbar/NavbarComputer/NavbarComputer.jsx

```diff
@@ -94,4 +93,8 @@ const Account = ({ user }) => {
   );
 };
 
-const UniverseHome = () => <Menu.Item href="/"><Logo /></Menu.Item>;
+const UniverseHome = () => (
+  <Menu.Item href="/">
+    <Logo />
+  </Menu.Item>
+);
```

Comment:
This file is missing a bunch of i18n as well. The non-internationalized strings here weren't added in this PR, so it's your choice whether to consider those fixes as being in scope for your ticket.

Reply:
At this time we aren't using this component for desktop. When we decide to switch to the milkyway navbar for desktop i'll make those changes.